docs/user/supported_devices.rst

```diff
@@ -427,7 +427,9 @@ ramips-mt7621
 
 * ZyXEL
 
+  - Multy M1
```

Comment:
two entries? Why not something like
- `Multy M1 (WSM20)`
- `WSM20 (Multy M1)`
- `WSM20 - Multy M1`

haven't made my mind up what i would prefer and what is most in line with how it's done for other devices.

Reply:
Most alias devices are on seperate lines, although there are exceptions: Arcadyan `VGV7510KW22 (o2 Box 6431)`, Nexx `WT3020AD/F/H`)
The most in line could be `Multy M1 (WSM20)`, like Mikrotik `DISC Lite5 ac (RBDiscG-5acD)`